server.js

```diff
@@ -8,10 +8,10 @@ const favicon = require('serve-favicon');
 const compression = require('compression');
 
 const app = express(),
-  // options = { 
-  //   key: fs.readFileSync(__dirname + '/key.pem'),
-  //   cert: fs.readFileSync(__dirname + '/cert.pem')
-  // },
+   options = { 
```

Comment:
I don't think you have used the options anywhere in the code. By themself, they are not of much use.

Reply:
Ok

Reply:
Can you reset this to the original. I in works in a new config and your PR will help that significantly .

I will merge this afterwards :smile: